src/ddmd/errors.d

```diff
@@ -94,33 +94,36 @@ extern (C++) void deprecationSupplemental(const ref Loc loc, const(char)* format
 // Just print, doesn't care about gagging
 extern (C++) void verrorPrint(const ref Loc loc, COLOR headerColor, const(char)* header, const(char)* format, va_list ap, const(char)* p1 = null, const(char)* p2 = null)
 {
+    Console* con = cast(Console*)global.console;
     const p = loc.toChars();
-    if (global.params.color)
-        setConsoleColorBright(true);
+    if (con)
+        con.setColorBright(true);
```

Comment:
I think what @WalterBright is saying is that in order to make use of the null object pattern, backtick handling would need to be moved to inside the `Console` interface, instead of being handled on the outside. Since the backticks are in the semantic domain of the compiler, not colored console output, it would require a second abstraction layer on top of `Console`, i.e. it's just not worth it.